### ISSUE TITLE
UnusedUses.UnusedUse false positive on phpdoc method

### ISSUE THREAD

**Issue by ohartl** [slevomat/coding-standard#290]
```php
<?php
namespace Project/SubNamespace;

use Project/Foo;

/**
 * @method bool someMethod(Foo $foo)
 */
class Bar extends Baz
{
    // ...
}
```

Is being reported as `3 | ERROR | [x] Type Project/Foo is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)` , but `Foo` is used in the `@method` Annotation

**Comment by Majkl578** [comment]
You have to enable the `searchAnnotations` option: https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesunuseduses-

**Comment by kukulich** [comment]
The option will not help. The “@method” format is really not supported now, @VasekPurchart already reported it to me privatly. I’m going to fix it next week.

**Comment by Majkl578** [comment]
Interesting, because I tested it locally and it is not reported with `searchAnnotations=true`.

**Comment by kukulich** [comment]
Really strange :)

**Comment by iammattcoleman** [comment]
@kukulich @ohartl's example (with the namespace slashes corrected) passes on 4.4.4, but does not pass on newer versions. It also passes on 4.4.3, which is the version I'll be using for now due to this issue and #288.

**Comment by benijo** [comment]
Same trouble in `property-read` annotation - with `searchAnnotations=true`

```
use App\Model\Security\User;
use Nette\Bridges\ApplicationLatte\Template;

/**
 * @property-read Template $template
 * @property-read User $user
 */
abstract class BasePresenter extends Presenter
{
```

**Comment by kukulich** [comment]
@benijo Your bug should be fixed in `dev-master`.

**Comment by iammattcoleman** [comment]
@kukulich I just confirmed that the issue is fixed for me in `dev-master`.

**Comment by kukulich** [comment]
Support for `@method` annotation just landed in master.

**Comment by ohartl** [comment]
@Majkl578 Note that we had `searchAnnotations=true` too, and it still reported that way. 

Thanks @kukulich . I really appreciate the time you spend fixing this.
